pages/common/dune.md

```diff
@@ -0,0 +1,20 @@
+# dune
+
+> A build system for [OCaml](https://ocaml.org) programs.
```

Comment:
```suggestion
> A build system for OCaml programs.
```
This is what the more information link is for.

Reply:
I've removed the link. However, those were different links: the first link is to the OCaml programming language website, the "more information" one was to the website of the Dune build tool, which is a third-party project. I believe for less-than-mainstream tools a link to the language would not be useless.

Reply:
Most clients (and terminals) don't support markdown syntax for links. If the link is important we could add in it parentheses.